ui/src/components/Dashboard/index.jsx

```diff
@@ -167,7 +167,7 @@ export default class Dashboard extends Component {
     search: null,
     helpView: null,
     docs: false,
-    disableTitleFormatting: false,
+    disableTitleFormatting: true,
```

Comment:
Doing this will affect all views. Let's do it on the quickstart view only.

Reply:
Done